.github/workflows/workflows.json

```diff
@@ -48,20 +51,17 @@
   "healthcare/fhir",
   "healthcare/hl7v2",
   "kms",
-  "game-servers/snippets",
   "mediatranslation",
   "monitoring/opencensus",
   "monitoring/prometheus",
   "monitoring/snippets",
-  "datacatalog/cloud-client",
-  "datacatalog/quickstart",
-  "datastore/functions",
   "service-directory/snippets",
   "scheduler",
   "speech",
   "talent",
   "translate",
   "video-intelligence",
-  "contact-center-insights",
+  "vision",
+  "vision/productSearch",
   "workflows"
 ]
```

Comment:
**Praise:** Thank you for sorting this in alphabetical order, Eric.

Reply:
👍🏻